getting-started/dc2-guide.rst

```diff
@@ -218,16 +219,21 @@ Processing Data with BPS
 
 The example data processing steps above assume a relatively small data volume, so running from the command line and using an sqlite APDB is appropriate.
 However, if you want to process larger data volumes, you'll need to use the Batch Processing System (BPS, :py:mod:`lsst.ctrl.bps`) and a PostgreSQL APDB.
-
-Describing how to set up a PostgreSQL APDB from scratch is beyond the scope of this guide.
 One key difference between using an sqlite APDB versus a PostgreSQL APDB is that the former is a file on disk created from scratch when running ``apdb-cli create-sql``.
-The latter requires a database to already exist, and ``apdb-cli create-sql`` turns the specified schema (via the ``namespace`` config option) in an existing PostgreSQL database into an empty APDB.
-As before, you will still need to run, e.g.,
+The latter requires a database to already exist and creation of the database is beyond the scope of this guide.
+
+As before, to create an empty sqlite APDB you will still need to run, e.g.,
 
 .. prompt:: bash
 
    apdb-cli create-sql sqlite:////path/to/my/database/apdb.sqlite3 apdb_config.py
 
+When working with a central PostgreSQL database (APDB), ``apdb-cli create-sql`` turns the specified schema (via the ``namespace`` config option) in an existing PostgreSQL database (``usdf-prompt-processing-dev`` in this example) into an empty APDB. To create a PostgreSQL APDB for a BPS configuration file that runs ``ApPipe.yaml``, these arguments are instead required to be passed into ``apdb-cli``:
```

Comment:
Is the "empty APDB" part actually true? I thought you needed to pass `--drop` for that to happen.

Reply:
Please write [one sentence per line](https://developer.lsst.io/restructuredtext/style.html#text-wrapping), it makes future diffs much clearer.